virt/release_notes/virt-4-15-release-notes.adoc

```diff
@@ -182,6 +182,13 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 
 [id="virt-4-15-bug-fixes"]
 == Bug fixes
+* Previously, EFI pipelines failed when started with a storage class that had the `volumeBindingMode` set to `WaitForFirstConsumer`. This fix removes the annotation in the `StorageClass` object that was causing the pipelines to fail. (link:https://issues.redhat.com/browse/CNV-32287[*CNV-32287*])
```

Comment:
@sjhala-ccs @dominikholler this is old news, but I don't know what are `EFI pipelines`. Can a URL be added?

Reply:
```suggestion
* Previously, the pipeline windows-efi-installer failed when started with a storage class that had the `volumeBindingMode` set to `WaitForFirstConsumer`. This fix removes the annotation in the `StorageClass` object that was causing the pipelines to fail. (link:https://issues.redhat.com/browse/CNV-32287[*CNV-32287*])
```